packages/app/components/FromToStackTokenPair.tsx

```diff
@@ -12,22 +14,27 @@ export const FromToStackTokenPair = ({
   fromToken,
   toToken,
   order,
```

Comment:
It seems to me that this component is receiving too much stuff. It should receive `to` and `from` token only. The logic should live in the parent component. If we have another source of tokens, should we add it here? If we want to have a component that receives an order we must create `OrderFromToStackTokenPair`.

Reply:
Makes sense, will try to simplify this component.

Reply:
fixed.